client/scss/objects/_avatar.scss

```diff
@@ -37,9 +37,5 @@
 
   &.square {
     border-radius: 0;
-
-    &:before {
-      border-radius: 0;
-    }
```

Comment:
Does nothing, not sure if there was an icon there before or if this is just dead code.